enhancements/network-observability.md

```diff
@@ -0,0 +1,367 @@
+
+The UI uses [Grafana](https://github.com/grafana/grafana)
+and [PatternFly](https://github.com/patternfly/patternfly) to provide a
+rich UI experience.
+
+
+### Installation and Packaging
+
+Network Observability will be delivered as an operator install which
+is available immediately when you access OpenShift.  Since it is part
+of OLM, it is not necessarily tied to a particular OCP release, and
+can be updated independently.
+
+A few resource-related parameters can be changed, but all values will
+have some default so that enabling network observability will be as
+simple as possible.  For more details, see the enhancement on Network
+Observability - Installation and Packaging,
+
+
+### Test Plan
+
+End-to-end testing will be done at different loads representing at least
+three personas in terms of nodes per cluster, pods per nodes, cores per
+cluster, and cores/nodes per cluster.  It will represent customers from
+different sectors of the industry such as in financial, healthcare, and
+technology running on different infrastructures (e.g. bare metal, AWS, GCP,
+Azure, IBM Cloud).  In the first release, it will target the lower end
+with the intention to scale up in subsequent releases.
+
+These are the criteria and metrics that will be measured.
+- Test with 5 nodes, 10 nodes, 25 nodes and 50 nodes in a cluster
+- Support at least 20K NetFlows per second
+- Document behavior when system is overwhelmed
+- Ensure all UI response times are reasonable
+- Test at different sampling rates and no sampling
+
+End-to-end automation testing will be done using [Cypress](https://www.cypress.io)
+for UI testing and [Ginkgo](https://onsi.github.io/ginkgo/) for backend
+testing.
+
+
+### Graduation Criteria
+
+The first release will be a Dev Preview.  The general acceptance criteria
+are:
+
+- Meet scalability targets
+- Complete end-to-end testing
+- No blockers in Jira
+- Do minimum stress testing
+
+Similar criteria will be apply to Tech Preview and GA but with
+different metrics such as increased node support and NetFlows
+per second.
+
+### Upgrade / Downgrade Strategy
+
+In a brown field scenario, if the customer is running OpenShift
+SDN, a change to OVN-Kubernetes is required.  Migrating to
+OVN-Kubernetes is a manual process that requires some downtime.  For
+more information, see [Migration to the OVN-Kubernetes network provider](https://docs.openshift.com/container-platform/4.8/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html).
+
+### Version Skew Strategy
+
+N/A since this is the first release
+
+## Implementation History
+
+TBD
+
+## Drawbacks
+
+Measuring ROI on network observability is difficult so it might be hard
+to justify the cost and resources to deploy this.  It attempts to find
+and resolve issues that you might not know exists.  The value you get
+may not be obvious because it is hard to know how much you saved by
+preventing ransomware or by allocating the right amount of resources
```

Comment:
"preventing ransomware" was a bit out of left field for me, since security isn't really a focus throughout the enhancement